### PR TITLE
Ahodges/RelNotes 20150610

### PR DESCRIPTION
**Pull request by lamagnifica**
@explorerleslie , @griffresch , @dan-f I tagged you for specific files with content for your review
@mhoeber , for your review, especially the documentation file

### CODE REVIEW

File: en_us/release_notes/source/2015/analytics/analytics_0610_2015.rst [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/analytics/analytics_0610_2015.rst:1]: @explorerleslie

File: en_us/release_notes/source/2015/documentation/doc_0610_2015.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
+==================================
+Documentation for edX Partners
+==================================
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/documentation/doc_0610_2015.rst:3]: @mhoeber

File: en_us/release_notes/source/2015/lms/lms_0610_2015.rst [hunk 3]
```diff
@@ -0,0 +1,26 @@
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/lms/lms_0610_2015.rst:1]: @explorerleslie , @mhoeber

File: en_us/release_notes/source/2015/studio/studio_0610_2015.rst [hunk 4]
```diff
@@ -0,0 +1,21 @@
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/studio/studio_0610_2015.rst:1]: @dan-f , @explorerleslie

File: en_us/release_notes/source/coming_soon.rst [hunk 5]
```diff
@@ -4,7 +4,14 @@ Coming Soon
 
 The following list describes features that are currently in development and
 that edX expects to release in the near future. After these features are
-released, we will add a link to related documentation.
+released, the release notes will include a link to related documentation.
+
```
Comment (lamagnifica) [inline en_us/release_notes/source/coming_soon.rst:8]: @griffresch

**Comment by explorerleslie** [comment]
:+1:

File: en_us/release_notes/source/2015/documentation/doc_0610_2015.rst [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+Building and Running an Open edX Course
+****************************************
+
+* The new `Setting Up Course Certificates`_ section describes how you
+  configure certificates for a course in Studio.
+
+* The new `Manage Course Fees`_ section describes configuration and
+  management of fees for paid courses.
+
+* The "Creating Exercises and Tools" section now includes a list of the
```
Comment (mhoeber) [inline en_us/release_notes/source/2015/documentation/doc_0610_2015.rst:54]: Do we not need this for partners?

**Comment by mhoeber** [comment]
looks good :+1:

**Comment by mhoeber** [comment]
can you add new Profile Images API doc to doc change list and link to:  http://edx.readthedocs.org/projects/edx-platform-api/en/latest/profile_images/index.html#profile-images-api-version-1-0

File: en_us/release_notes/source/2015/documentation/doc_0610_2015.rst (hunk 6 above)
Comment (lamagnifica) [inline en_us/release_notes/source/2015/documentation/doc_0610_2015.rst:54]: No, we are not rolling out the assessments or discussions on the edX mobile apps for edx.org courses until we have done a lot more performance testing -- expected in July 2015

File: en_us/release_notes/source/2015/documentation/doc_0610_2015.rst [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+* The section about `Certificates`_ now includes information about
+  sharing certificates and digital badges.
+
+* The new `Student Notes`_ section describes how learners can highlight and
+  make notes about course content.
+
+* The new `Bookmarks`_  section describes how learners can mark course content
+  and return to it later.
+
+========================
```
Comment (lamagnifica) [inline en_us/release_notes/source/2015/documentation/doc_0610_2015.rst:77]: @mhoeber added this vv

File: en_us/release_notes/source/2015/documentation/doc_0610_2015.rst [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+* The new `Searching the Course`_ section describes how learners can search for
+  a term in a single course or in all of the courses they are enrolled in.
+
+* The section about `Certificates`_ now includes information about
+  sharing certificates and digital badges.
+
+* The new `Student Notes`_ section describes how learners can highlight and
+  make notes about course content.
+
+* The new `Bookmarks`_  section describes how learners can mark course content
```
Comment (explorerleslie) [inline en_us/release_notes/source/2015/documentation/doc_0610_2015.rst]: Hi @lamagnifica just noticing this: why is bookmarks listed here? It hasn't been released yet. @srpearce

Comment (lamagnifica) [inline en_us/release_notes/source/2015/documentation/doc_0610_2015.rst]: It is listed in the change log as going yesterday...

Comment (srpearce) [inline en_us/release_notes/source/2015/documentation/doc_0610_2015.rst]: It's going into the Open edX Learners Guide, not Partners - or has it not released at all, even for Open edX? I thought we were just working on the API stuff right now but that everything else had merged.

Comment (explorerleslie) [inline en_us/release_notes/source/2015/documentation/doc_0610_2015.rst]: It hasn't merged yet for open edX, @srpearce

File: en_us/release_notes/source/2015/studio/studio_0610_2015.rst (hunk 4 above)
Comment (dan-f) [inline en_us/release_notes/source/2015/studio/studio_0610_2015.rst:1]: :+1: lgtm!

**Pull request merged**